### ISSUE TITLE
Outlook / Office 365 Calendar: Reoccurring meetings aren't being imported

### ISSUE THREAD

**Issue by miketdn** [sawhney17/logseq-calendars-plugin#49]
Great plugin! I've been able to connect my calendar and import events the way I like it, however it doesn't seem to import all events from my calendar. 

The pattern I'm seeing is that the ones that are not imported are reoccurring meetings (part of a series). The one off events are being imported just fine.

Has anyone else experienced this issue? Is there something in the configuration I need to change?

**Comment by kennethaar** [comment]
Same issue here. I think it is a known issue. Maybe a bug in the underlying framework.

https://github.com/jens-maus/node-ical/issues/113

Sadly I have no idea how to fix it.

**Comment by kennethaar** [comment]
Probably same as #26 

Reoccurring meetings from others seem to be problematic.

Workaround could be to copy event and invite yourself to have it show up.

**Comment by LuukSuurmeijer** [comment]
@kennethaar I have tried this but it seems like all recurring events are not being imported to me, also the ones I created myself.

**Comment by kennethaar** [comment]
Yeah, it stopped working for me to. I have been playing around with PowerAutomate to make it add all appointments directly to Logseq. But that also seems a little to frail for my liking.

**Comment by kennethaar** [comment]
Managed to get PowerAutomate to work so now I finally get recurring items into my daily journal. Another advantage is that I can get any information I want from the calendar into Logseq. Like the attendees in a meeting and link to the calendar entry. Only issue is that I can't make the file dynamic. So I go via a file called calendar.md instead. Sadly it takes way too long to set up. Mostly since Power Automate is so bad at automation and has lots of annoying bugs. If you aren't locked into MS then I would recommend Zapier or IFTTT instead.

This is the convoluted mess I ended up with:

![image](https://github.com/sawhney17/logseq-calendars-plugin/assets/5931199/c2a7e611-ba50-43f1-b2f5-0fd02d7e8b6e)
